README.md

```diff
@@ -55,6 +55,21 @@ yarn start
 
 The above command will run a web server on port :5173 that will serve the UI assets, with hot reloading support, and it will proxy all API requests to `http://localhost:16686` where Jaeger query should be running.
 
+------------
+### Using Windows OS for Development?
```

Comment:
it would more sense to have this as the 2nd subsection under Development. And the 1st subsection should be Prerequisites that lists the requirements, namely `nvm`, Node.js + `npm`, and `yarn` (don't think we need anything else).